src/hdmf/utils.py

```diff
@@ -634,19 +634,19 @@ def __get_shape_helper(local_data):
             if len(local_data) and not isinstance(local_data[0], (str, bytes)):
                 shape.extend(__get_shape_helper(local_data[0]))
         return tuple(shape)
+
+    shape = np.shape(data)
+    if shape:
+        return shape
     if hasattr(data, 'maxshape'):
         return data.maxshape
-    elif hasattr(data, 'shape'):
+    if hasattr(data, 'shape'):
```

Comment:
This is redundant with `np.shape`. See https://github.com/numpy/numpy/blob/v1.18.4/numpy/core/fromnumeric.py#L1903-L1946

Reply:
never mind